### PR TITLE
jenkinsfiles: Don't display nulls in current build display name

### PR DESCRIPTION
**Pull request by twpayne**
Signed-off-by: Tom Payne <tom@isovalent.com>

### CODE REVIEW

File: jenkinsfiles/ginkgo-gke.Jenkinsfile [hunk 1]
```diff
@@ -30,12 +30,16 @@ pipeline {
 
     stages {
         stage('Set build name') {
-            when {
-                not {environment name: 'GIT_BRANCH', value: 'origin/master'}
-            }
```
Comment (nbusseneau) [inline jenkinsfiles/ginkgo-gke.Jenkinsfile]: Just for the sake of it, I've tried with:
```
when {
    not {
        anyOf {
            environment name: 'ghprbPullTitle', value: null
            environment name: 'ghprbPullLink', value: null
        }
    }
}
```
and it does compare against `null` values. I personally think this looks cleaner, what do you think? 🤔

Comment (twpayne) [reply jenkinsfiles/ginkgo-gke.Jenkinsfile]: Agreed, PR updated.

**Review by nbusseneau — APPROVED** [review]
🚀

**Comment by nbusseneau** [comment]
Note; this will have to be backported if we want it to apply where the jobs are currently displaying `null` values (which is older stable branches).

**Pull request merged**